### PR TITLE
Update Broadcom SAI version to 6.1

### PR DESCRIPTION
**Pull request by tjchadaga**
#### Why I did it
Update Broadcom SAI version to 6.1.0.3. Currently built with Buster.

#### How I did it
Includes change to use integrated SDK (no separate patch files)
Main fixes include CS00012228728, CS00012232402, CS00012231916, CS00012221417.

#### How to verify it

**Basic validation done on DNX and fdb/test_fdb.py test**

```
bsv
BRCM SAI ver: [6.1.0.3], OCP SAI ver: [1.9.1], SDK ver: [sdk-6.5.24]


admin@str2-a7280cr3-4:~$ show ip bgp sum

IPv4 Unicast Summary:
BGP router identifier 10.1.0.32, local AS number 65100 vrf-id 0
BGP table version 6407
RIB entries 12807, using 2458944 bytes of memory
Peers 4, using 87264 KiB of memory
Peer groups 4, using 256 bytes of memory


Neighbhor      V     AS    MsgRcvd    MsgSent    TblVer    InQ    OutQ  Up/Down      State/PfxRcd  NeighborName
-----------  ---  -----  ---------  ---------  --------  -----  ------  ---------  --------------  --------------
10.0.0.57      4  64600       3396       3399         0      0       0  00:09:35             6400  ARISTA01T1
10.0.0.59      4  64600       3399       3398         0      0       0  00:09:35             6400  ARISTA02T1
10.0.0.61      4  64600       3396       3397         0      0       0  00:09:35             6400  ARISTA03T1
10.0.0.63      4  64600       3399       3397         0      0       0  00:09:35             6400  ARISTA04T1

Total number of neighbors 4
admin@str2-a7280cr3-4:~$ 
admin@str2-a7280cr3-4:~$ 
admin@str2-a7280cr3-4:~$ show ip route sum
Route Source         Routes               FIB  (vrf default)
connected            7                    7                    
ebgp                 6403                 6403                 
ibgp                 0                    0                    
------
Totals               6410                 6410                 


admin@str2-a7280cr3-4:~$ 

admin@str2-a7280cr3-4:~$ show ipv6 bgp summary 

IPv6 Unicast Summary:
BGP router identifier 10.1.0.32, local AS number 65100 vrf-id 0
BGP table version 6407
RIB entries 12807, using 2458944 bytes of memory
Peers 4, using 87264 KiB of memory
Peer groups 4, using 256 bytes of memory


Neighbhor      V     AS    MsgRcvd    MsgSent    TblVer    InQ    OutQ  Up/Down      State/PfxRcd  NeighborName
-----------  ---  -----  ---------  ---------  --------  -----  ------  ---------  --------------  --------------
fc00::7a       4  64600       3403       3407         0      0       0  00:09:59             6400  ARISTA03T1
fc00::7e       4  64600       3404       3407         0      0       0  00:09:59             6400  ARISTA04T1
fc00::72       4  64600       3403       3407         0      0       0  00:10:00             6400  ARISTA01T1
fc00::76       4  64600       3404       3407         0      0       0  00:10:00             6400  ARISTA02T1

Total number of neighbors 4
admin@str2-a7280cr3-4:~$

admin@str2-a7280cr3-4:~$ show interfaces status | grep up
     Ethernet4          2,3     100G   9100     rs   Ethernet2/1           trunk      up       up  QSFP28 or later         off
     Ethernet8          4,5     100G   9100     rs   Ethernet3/1           trunk      up       up  QSFP28 or later         off
    Ethernet12          6,7     100G   9100     rs   Ethernet4/1           trunk      up       up  QSFP28 or later         off
    Ethernet16          8,9     100G   9100     rs   Ethernet5/1           trunk      up       up  QSFP28 or later         off
    Ethernet20        10,11     100G   9100     rs   Ethernet6/1           trunk      up       up  QSFP28 or later         off
    Ethernet24        12,13     100G   9100     rs   Ethernet7/1           trunk      up       up  QSFP28 or later         off
    Ethernet28        14,15     100G   9100     rs   Ethernet8/1           trunk      up       up  QSFP28 or later         off
    Ethernet32        16,17     100G   9100     rs   Ethernet9/1           trunk      up       up  QSFP28 or later         off
    Ethernet36        18,19     100G   9100     rs  Ethernet10/1           trunk      up       up  QSFP28 or later         off
    Ethernet40        20,21     100G   9100     rs  Ethernet11/1           trunk      up       up  QSFP28 or later         off
    Ethernet44        22,23     100G   9100     rs  Ethernet12/1           trunk      up       up  QSFP28 or later         off
    Ethernet48        24,25     100G   9100     rs  Ethernet13/1           trunk      up       up  QSFP28 or later         off
    Ethernet52        26,27     100G   9100     rs  Ethernet14/1           trunk      up       up  QSFP28 or later         off
    Ethernet56        28,29     100G   9100     rs  Ethernet15/1           trunk      up       up  QSFP28 or later         off
    Ethernet60        30,31     100G   9100     rs  Ethernet16/1           trunk      up       up  QSFP28 or later         off
    Ethernet64        72,73     100G   9100     rs  Ethernet17/1           trunk      up       up  QSFP28 or later         off
    Ethernet68        74,75     100G   9100     rs  Ethernet18/1           trunk      up       up  QSFP28 or later         off
    Ethernet72        76,77     100G   9100     rs  Ethernet19/1           trunk      up       up  QSFP28 or later         off
    Ethernet76        78,79     100G   9100     rs  Ethernet20/1           trunk      up       up  QSFP28 or later         off
    Ethernet80        64,65     100G   9100     rs  Ethernet21/1           trunk      up       up  QSFP28 or later         off
    Ethernet84        66,67     100G   9100     rs  Ethernet22/1           trunk      up       up  QSFP28 or later         off
    Ethernet88        68,69     100G   9100     rs  Ethernet23/1           trunk      up       up  QSFP28 or later         off
    Ethernet92        70,71     100G   9100     rs  Ethernet24/1           trunk      up       up  QSFP28 or later         off
    Ethernet96        56,57     100G   9100     rs  Ethernet25/1           trunk      up       up  QSFP28 or later         off
   Ethernet112        48,49     100G   9100     rs  Ethernet29/1  PortChannel101      up       up  QSFP28 or later         off
   Ethernet116        50,51     100G   9100     rs  Ethernet30/1  PortChannel102      up       up  QSFP28 or later         off
   Ethernet120        52,53     100G   9100     rs  Ethernet31/1  PortChannel103      up       up  QSFP28 or later         off
   Ethernet124        54,55     100G   9100     rs  Ethernet32/1  PortChannel104      up       up  QSFP28 or later         off
PortChannel101          N/A     100G   9100    N/A           N/A          routed      up       up              N/A         N/A
PortChannel102          N/A     100G   9100    N/A           N/A          routed      up       up              N/A         N/A
PortChannel103          N/A     100G   9100    N/A           N/A          routed      up       up              N/A         N/A
PortChannel104          N/A     100G   9100    N/A           N/A          routed      up       up              N/A         N/A
admin@str2-a7280cr3-4:~$ 

```
MACSec validation done by @judyjoseph 

**Basic XGS validation:**   @gechiang - please update as needed

Tests run:
```
     fib/test_fib.py
     vxlan/test_vxlan_decap.py
     fdb/test_fdb.py
     decap/test_decap.py
     ipfwd/test_dir_bcast.py
``` 

```
bsv
BRCM SAI ver: [6.1.0.3], OCP SAI ver: [1.9.1], SDK ver: [sdk-6.5.24]
drivshell>

admin@str2-dx010-acs-7:~$ show ip bgp sum

IPv4 Unicast Summary:
BGP router identifier 10.1.0.32, local AS number 65100 vrf-id 0
BGP table version 6405
RIB entries 12807, using 2458944 bytes of memory
Peers 4, using 87264 KiB of memory
Peer groups 4, using 256 bytes of memory


Neighbhor      V     AS    MsgRcvd    MsgSent    TblVer    InQ    OutQ  Up/Down      State/PfxRcd  NeighborName
-----------  ---  -----  ---------  ---------  --------  -----  ------  ---------  --------------  --------------
10.0.0.57      4  64600       3353       3354         0      0       0  00:07:24             6400  ARISTA01T1
10.0.0.59      4  64600       3356       3354         0      0       0  00:07:24             6400  ARISTA02T1
10.0.0.61      4  64600       3353       3354         0      0       0  00:07:24             6400  ARISTA03T1
10.0.0.63      4  64600       3353       3354         0      0       0  00:07:24             6400  ARISTA04T1

Total number of neighbors 4
admin@str2-dx010-acs-7:~$

admin@str2-dx010-acs-7:~$ show ip route sum
Route Source         Routes               FIB  (vrf default)
connected            7                    7                    
ebgp                 6403                 6403                 
ibgp                 0                    0                    
------
Totals               6410                 6410                 


admin@str2-dx010-acs-7:~$ 

admin@str2-dx010-acs-7:~$ show ipv6 bgp sum

IPv6 Unicast Summary:
BGP router identifier 10.1.0.32, local AS number 65100 vrf-id 0
BGP table version 6405
RIB entries 12807, using 2458944 bytes of memory
Peers 4, using 87264 KiB of memory
Peer groups 4, using 256 bytes of memory


Neighbhor      V     AS    MsgRcvd    MsgSent    TblVer    InQ    OutQ  Up/Down      State/PfxRcd  NeighborName
-----------  ---  -----  ---------  ---------  --------  -----  ------  ---------  --------------  --------------
fc00::7a       4  64600       3368       3371         0      0       0  00:08:09             6400  ARISTA03T1
fc00::7e       4  64600       3368       3371         0      0       0  00:08:09             6400  ARISTA04T1
fc00::72       4  64600       3368       3371         0      0       0  00:08:09             6400  ARISTA01T1
fc00::76       4  64600       3368       3371         0      0       0  00:08:09             6400  ARISTA02T1

Total number of neighbors 4
admin@str2-dx010-acs-7:~$ 

admin@str2-dx010-acs-7:~$ show interfaces status | grep up
     Ethernet4      69,70,71,72     100G   9100     rs     etp2           trunk      up       up  QSFP28 or later         off
     Ethernet8      73,74,75,76     100G   9100     rs     etp3           trunk      up       up  QSFP28 or later         off
    Ethernet12      77,78,79,80     100G   9100     rs     etp4           trunk      up       up  QSFP28 or later         off
    Ethernet16      33,34,35,36     100G   9100     rs     etp5           trunk      up       up  QSFP28 or later         off
    Ethernet20      37,38,39,40     100G   9100     rs     etp6           trunk      up       up  QSFP28 or later         off
    Ethernet24      41,42,43,44     100G   9100     rs     etp7           trunk      up       up  QSFP28 or later         off
    Ethernet28      45,46,47,48     100G   9100     rs     etp8           trunk      up       up  QSFP28 or later         off
    Ethernet32      49,50,51,52     100G   9100     rs     etp9           trunk      up       up  QSFP28 or later         off
    Ethernet36      53,54,55,56     100G   9100     rs    etp10           trunk      up       up  QSFP28 or later         off
    Ethernet40      57,58,59,60     100G   9100     rs    etp11           trunk      up       up  QSFP28 or later         off
    Ethernet44      61,62,63,64     100G   9100     rs    etp12           trunk      up       up  QSFP28 or later         off
    Ethernet48      81,82,83,84     100G   9100     rs    etp13           trunk      up       up  QSFP28 or later         off
    Ethernet52      85,86,87,88     100G   9100     rs    etp14           trunk      up       up  QSFP28 or later         off
    Ethernet56      89,90,91,92     100G   9100     rs    etp15           trunk      up       up  QSFP28 or later         off
    Ethernet60      93,94,95,96     100G   9100     rs    etp16           trunk      up       up  QSFP28 or later         off
    Ethernet64     97,98,99,100     100G   9100     rs    etp17           trunk      up       up  QSFP28 or later         off
    Ethernet68  101,102,103,104     100G   9100     rs    etp18           trunk      up       up  QSFP28 or later         off
    Ethernet72  105,106,107,108     100G   9100     rs    etp19           trunk      up       up  QSFP28 or later         off
    Ethernet76  109,110,111,112     100G   9100     rs    etp20           trunk      up       up  QSFP28 or later         off
    Ethernet80          1,2,3,4     100G   9100     rs    etp21           trunk      up       up  QSFP28 or later         off
    Ethernet84          5,6,7,8     100G   9100     rs    etp22           trunk      up       up  QSFP28 or later         off
    Ethernet88       9,10,11,12     100G   9100     rs    etp23           trunk      up       up  QSFP28 or later         off
    Ethernet92      13,14,15,16     100G   9100     rs    etp24           trunk      up       up  QSFP28 or later         off
    Ethernet96      17,18,19,20     100G   9100     rs    etp25           trunk      up       up  QSFP28 or later         off
   Ethernet112  113,114,115,116     100G   9100     rs    etp29  PortChannel101      up       up  QSFP28 or later         off
   Ethernet116  117,118,119,120     100G   9100     rs    etp30  PortChannel102      up       up  QSFP28 or later         off
   Ethernet120  121,122,123,124     100G   9100     rs    etp31  PortChannel103      up       up  QSFP28 or later         off
   Ethernet124  125,126,127,128     100G   9100     rs    etp32  PortChannel104      up       up  QSFP28 or later         off
PortChannel101              N/A     100G   9100    N/A      N/A          routed      up       up              N/A         N/A
PortChannel102              N/A     100G   9100    N/A      N/A          routed      up       up              N/A         N/A
PortChannel103              N/A     100G   9100    N/A      N/A          routed      up       up              N/A         N/A
PortChannel104              N/A     100G   9100    N/A      N/A          routed      up       up              N/A         N/A
admin@str2-dx010-acs-7:~

```
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111

### CODE REVIEW

**Review by gechiang — DISMISSED** [review]
LGTM

File: platform/broadcom/saibcm-modules/debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+opennsl (6.1.0.3) unstable; urgency=medium
```
Comment (judyjoseph) [inline platform/broadcom/saibcm-modules/debian/changelog:1]: Can you do a similar change in https://github.com/Azure/saibcm-modules/commits/sdk-6.5.24-gpl-dnx.
Also update the submodule in platform/broadcom/saibcm-modules-dnx

Comment (tjchadaga) [reply platform/broadcom/saibcm-modules/debian/changelog:1]: Done

**Review by judyjoseph — APPROVED** [review]

**Comment by zhengbiqing** [comment]
"Update Broadcom SAI version to 6.1.0.3"     Can you tell me where is the "Broadcom SAI"?  thanks!

**Pull request merged**